search/feature_offset_match.hpp

```diff
@@ -196,6 +196,14 @@ class TrieValuesHolder
 template <typename DFA>
 struct SearchTrieRequest
 {
+  SearchTrieRequest() = default;
```

Comment:
Interesting, how did it work without the constructor before?..

Reply:
Compiler implicitly declares defaulted destructor if user doesn't declare it explicitly.
The real change is explicit move constructor.